src/sdg/producten/tests/test_update_catalogs.py

```diff
@@ -1,27 +1,327 @@
+from datetime import date, timedelta
+
+from django.core.management import call_command
+from django.db.models import Q
 from django.test import TestCase
 
+from sdg.core.models import ProductenCatalogus
+from sdg.core.tests.factories.catalogus import ProductenCatalogusFactory
+from sdg.organisaties.tests.factories.overheid import LokaleOverheidFactory
+from sdg.producten.models.localized import LocalizedProduct
+from sdg.producten.models.product import Product, ProductVersie
+from sdg.producten.tests.factories.localized import LocalizedProductFactory
+from sdg.producten.tests.factories.product import (
+    ProductVersieFactory,
+    ReferentieProductFactory,
+)
+
 
 class UpdateCatalogTests(TestCase):
     def test_create_catalog_from_reference_if_enabled(self):
-        pass
+        producten_catalogus_table = ProductenCatalogus.objects.all()
+
+        ref_overheid = LokaleOverheidFactory.create(
+            automatisch_catalogus_aanmaken=False
+        )
+        ProductenCatalogusFactory.create(
+            lokale_overheid=ref_overheid,
+            is_referentie_catalogus=True,
+        )
+
+        LokaleOverheidFactory.create(
+            automatisch_catalogus_aanmaken=True,
+            organisatie__owms_end_date=None,
+        )
+
+        self.assertEqual(producten_catalogus_table.count(), 1)
+        call_command("update_catalogs")
+        self.assertEqual(producten_catalogus_table.count(), 2)
 
```

Comment:
Also, add some more checks on the created catalog, thus (pseudo code!):
```suggestion
new_catalog = ProductenCatalogus.objects.get(is_referentie_catalogus=False)  # There's only 1 in this test
self.assertEqual(new_catalog.lokale_overheid, expected_lokale_overheid)
self.assertEqual(new_catalog.versie, ref_catalog.versie)
self.assertEqual(new_catalog.naam, expected_name)
# etc...
```

What might seem obvious now, might break in the future.